mage/ginkgo/lib.go

```diff
@@ -369,3 +367,27 @@ func (ns NsWrapper) HelmInstall(chartPath string) {
 	err := helm_util.Install(chartPath, "cass-operator", ns.Namespace, overrides)
 	mageutil.PanicOnError(err)
 }
+
+// Note that the actual value will be cast to a string before the comparison with the expectedValue
+func (ns NsWrapper) ExpectKeyValue(m map[string]interface{}, key string, expectedValue string) {
```

Comment:
ExpectKeyValues(), which takes a whole map, and the helper function ExpectKeyValue() are a first-pass at a generic way to validate key/values.  Right now everything is cast to a string.  I added a helper for dealing with Float64 values because the Ports from the service were coming back as Float64s for some reason.  I am open to suggestion for improvements, but we may want to handle them in a follow-up ticket.